widgets/src/main/java/org/pentaho/gwt/widgets/client/dialogs/DialogBox.java

```diff
@@ -534,15 +538,48 @@ private void setDomModal( boolean modal ) {
   }
   // endregion
 
-  public boolean onKeyDownPreview( char key, int modifiers ) {
-    // Use the popup's key preview hooks to close the dialog when either
-    // enter or escape is pressed.
-    switch ( key ) {
-      case KeyboardListener.KEY_ESCAPE:
-        hide();
```

Comment:
`onKeyDownPreview` is deprecated. This code for handling the ESC key was integrated in the expected alternative, the `onPreviewNativeEvent` method, where the new case for handling "clicking" is also being handled.